docs/root/intro/version_history.rst

```diff
@@ -12,6 +12,7 @@ Version history
 * tcp_proxy: added :ref:`ClusterWeight.metadata_match<envoy_api_field_config.filter.network.tcp_proxy.v2.TcpProxy.WeightedCluster.ClusterWeight.metadata_match>`
 * tcp_proxy: added :ref:`hash_policy<envoy_api_field_config.filter.network.tcp_proxy.v2.TcpProxy.hash_policy>`
 * tls: remove TLS 1.0 and 1.1 from client defaults
+* http: add support for encoding http1 trailers. To disable use :ref:`disable_trailers <envoy_api_field.core.Http1ProtocolOptions.disable_trailers>`.
```

Comment:
I am failing the doc check here due to 
```
/source/generated/rst/intro/version_history.rst:15: WARNING: undefined label: envoy_api_field.core.http1protocoloptions.disable_trailers (if the link has no caption the label must precede a section header)
```

Which I have no idea what that means...

Reply:
You are using the wrong label for the auto-generated reference. 

I think you want `envoy_api_field_core.Http1ProtocolOptions.disable_trailers`